packages/j-mail/src/handler.ts

```diff
@@ -121,20 +134,26 @@ export const sendUpdateEmails: Handler = async (event, context) => {
     const data = await getDataForUpdateEmail(userId)
     const body = updateEmail(data)
 
-    await enqueueEmail({
-      from: 'robin@journaly.com',
-      to: data.user.email,
-      subject: 'New Activity on Journaly 📖',
-      html: body,
-    })
+    // Currently we only support 'DAILY' OR 'OFF' so we simply
+    // skip enqueueEmail if 'OFF'.
+    // When we build support for 'WEEKLY' we'll need to make this
+    // logic a little more sophisticated.
+    if (data.userConfig.digestEmail !== DigestEmailConfiguration.OFF) {
```

Comment:
isn't `userConfig` nullable? Might need to optional chain here.

Since we are changing jmail (which is good!) we should probably deploy this to stage and confirm it works.